GliaWidgets/ViewController/Survey/Components/Survey.ValidationErrorView.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+
+extension Survey {
+    final class ValidationErrorView: View {
+        let validationMessage = UILabel().make {
+            $0.numberOfLines = 0
+            $0.textColor = .red
+            $0.font = .boldSystemFont(ofSize: 12)
+            $0.text = "Please provide an answer."
+        }
+        let validationImage = UIImageView().make {
+            $0.image = Asset.surveyValidationError.image
+        }
+        lazy var validationMessageHStack = UIStackView.make(.horizontal, spacing: 8)(
```

Comment:
Can't the Makeable protocol be used for stackviews?

Reply:
in general can be, but in case of Makeable protocol you will need to define manually for each UIStackView this code
```
$0.translatesAutoresizingMaskIntoConstraints = false
$0.axis = axis
$0.spacing = spacing
$0.distribution = distribution
$0.alignment = alignment

... // other customization
```
to avoid this situation, I just create a separate static function :) just useful

Reply:
I think it makes sense, but the issue here that `makeView` name does not _reflect_ that `translatesAutoresizingMaskIntoConstraints = true`, basically hides it. In my opinion it is better to be explicit about what method does under the hood. This can be addresses by having dedicated functions that do one thing and then composed into single one that will be applied to the view. Here's some good [examples](https://github.com/pointfreeco/episode-code-samples/blob/main/0003-styling-with-functions/Styling%20with%20Functions.playground/Pages/Complete.xcplaygroundpage/Contents.swift).
One note: since we need to release this feature in the near future we can leave it as is for now and then rework it.